hammer/reporting-remediation/reporting/create_ecs_external_image_source_issue_tickets.py

```diff
@@ -66,18 +66,6 @@ def create_tickets_ecs_external_images(self):
                             bu=bu, product=product,
                         )
                         IssueOperations.set_status_closed(ddb_table, issue)
-                    # issue.status != IssueStatus.Closed (should be IssueStatus.Open)
```

Comment:
Why this section was removed from this and five more modules?

Reply:
Here in this case, we don't have option to modify/change existing resources (no remediation also). So deleted these code blocks (which Oleksii suggested for as code review for previous issues) which are not required to validate and sent notification for change.